src/app/pages/camper/schedule/schedule.component.ts

```diff
@@ -1,15 +1,75 @@
 import { Component, OnInit } from '@angular/core';
+import { ScheduleService } from "../../../service/schedule.service";
+import { interval } from 'rxjs';
 
 @Component({
   selector: 'app-schedule',
   templateUrl: './schedule.component.html',
   styleUrls: ['./schedule.component.scss']
 })
 export class ScheduleComponent implements OnInit {
+	
+  	private todayDate;
+  	private schedulelist;
+  	private hours;
+  	private mins;
+  	private secs;
 
-  constructor() { }
+  constructor(
+  	public schService:ScheduleService
+  	) { }
 
   ngOnInit() {
+  	this.schService.getSheet().subscribe(response => {
+
+	  	var date = ["20 JULY", "21 JULY", "22 JULY"];
+
+	  	var scheduleArr = response.values; 
+	  	this.schedulelist = this.getSchedule(scheduleArr);
+
+		interval(1000).subscribe(n =>
```

Comment:
for lambda expression, 

if it is a single line it is usually in this form 

`foo(n => n * 2);` this is equivalent to `foo(function(n) { return n * 2 });`

for multiline lambda expressions, you need to wrap it in a `{ }` braces, for multiple arguments you need to wrap the parameters in `( )` brackets 

i.e. 
```javascript
foo((x, y, z) => {
    const k = x * y * z;
    return k + x / z;
});
```